src/utilities/theme/types.ts

```diff
@@ -22,14 +22,28 @@ export interface TopBar extends ThemeColor {
 }
 
 export type ThemeColors = {
-  topBar: TopBar;
+  topBar?: TopBar;
+  surface?: string;
+  onSurface?: string;
+  interactive?: string;
+  interactiveNeutral?: string;
+  branded?: string;
+  critical?: string;
+  warning?: string;
+  highlight?: string;
+  success?: string;
 };
 
-export interface Theme {
+export interface ThemeConfig {
   /** Sets the logo for the top bar and contextual save bar components*/
   logo?: ThemeLogo;
   /** Sets the background color of the top bar component. Complimentary and typography colors are determined programmatically */
   colors?: ThemeColors;
 }
 
+export interface Theme {
+  /** Sets the logo for the top bar and contextual save bar components*/
+  logo?: ThemeLogo;
+}
+
```

Comment:
Is there a way to make these type changes backwards compatible, or will this need to be shipped as part of a `v5.0.0` alpha release?

Reply:
Theme and the other types in here are never exposed publicly (check usage references and see what files they get exported from), you're free to name them whatever you wish. The public entry point for using these shapes would be plucking the type from `AppProviderProps['theme']` and we'll be keeping that same prop name. It doesn't matter that we're renaming the type that references internally from `Theme` to `ThemeConfig` because the shape will remain compatible with what it expects currently.